db/migrations/20200616090757-application-state.js

```diff
@@ -0,0 +1,59 @@
+'use strict';
+
+var dbm;
```

Comment:
Is it worth using `const` and `let` instead of `var` to differentiate between things that can subsequently be changed and things which are fixed?